src/pretix/plugins/sendmail/api.py

```diff
@@ -88,6 +90,11 @@ def validate(self, data):
                 ]:
                     raise ValidationError(f'status {s} not allowed: restrict_to_status may only include valid states')
 
+        checked_in_status = full_data.get('checked_in_status')
+        if checked_in_status not in [state[0] for state in Rule.CHECK_IN_STATUS_CHOICES]:
+            raise ValidationError(
```

Comment:
Is this not already validated by REST framework for a field with choices?

Reply:
Yes, yes it does. I now deleted these lines.